server/index.js

```diff
@@ -23,6 +23,7 @@ const open = require('open');
 const portfinder = require('portfinder');
 const crypto = require("crypto");
 const fse = require('fs-extra');
+const yaml = require("js-yaml");
```

Comment:
```suggestion
const yaml = require('js-yaml');
```
(To avoid mixing char/string delimiters).